mender-update/update_module/v3/update_module_test.cpp

```diff
@@ -132,3 +148,185 @@ TEST_F(UpdateModuleTests, DiscoverUpdateModulesNoExecutablesTest) {
 	auto modules = ex_modules.value();
 	EXPECT_EQ(modules.size(), 0);
 }
+
+testing::AssertionResult FileContains(const string &filename, const string &expected_content) {
```

Comment:
This could possible be the start of a more general testing utility library for us?